src/website/views/components/dropdown/_properties.pug

```diff
@@ -1,7 +1,8 @@
-h2 JavaScript
-h3 Properties
+h2 Web Component
+:stencilDoc(name='chi-dropdown')
 
```

Comment:
You can follow this example where we have all documented: 

https://assets.ctl.io/chi/5.16.0/components/drawer/?theme=Portal#web-component

As you can see the Web components, Vue and Javascript have their own properties, methods and events so in the dropdown we need to follow the same pattern.

The `stencilDoc` will automatically add the properties, methods and events for the web components, but we still need the Javascript part.